pkg/ddl/backfilling_scheduler.go

```diff
@@ -472,16 +455,10 @@ func (b *ingestBackfillScheduler) createWorker(
 ) workerpool.Worker[IndexRecordChunk, workerpool.None] {
 	reorgInfo := b.reorgInfo
 	job := reorgInfo.Job
-	sessCtx, err := newSessCtx(reorgInfo.d.store, reorgInfo.ReorgMeta.SQLMode, reorgInfo.ReorgMeta.Location, reorgInfo.ReorgMeta.ResourceGroupName)
-	if err != nil {
-		b.sendResult(&backfillResult{err: err})
-		return nil
-	}
```

Comment:
`newAddIndexIngestWorker` is only called by `createWorker`, so we moved the context construction to ``. It is equivalent to the previous code.

Reply:
Yes, LGTM